fairing/builders/append/append.py

```diff
@@ -29,26 +29,30 @@
 
 DEFAULT_IMAGE_NAME = 'fairing-job'
 DEFAULT_BASE_IMAGE = 'gcr.io/kubeflow-images-public/fairing:dev'
-DEFAULT_REGISTRY   = 'index.docker.io'
+DEFAULT_REGISTRY = 'index.docker.io'
 
 TEMP_TAR_GZ_FILENAME = '/tmp/fairing.layer.tar.gz'
 _THREADS = 8
 
+
 class AppendBuilder(BuilderInterface):
     def __init__(self,
-                repository,
-                image_name=DEFAULT_IMAGE_NAME,
-                base_image=DEFAULT_BASE_IMAGE,
-                notebook_file=None,
-                image_tag=None):
+                 repository,
+                 image_name=DEFAULT_IMAGE_NAME,
+                 base_image=DEFAULT_BASE_IMAGE,
+                 notebook_file=None,
+                 image_tag=None,
+                 resource_limits=None):
         self.repository = repository
         self.image_name = image_name
         self.base_image = base_image
         self.image_tag = image_tag
         self.notebook_file = notebook_file
+        self.resource_limits = resource_limits
```

Comment:
I added a new resource_limits argument to require a training job gets a gpu.  Not sure if a higher level abstraction is appropriate...